README.md

```diff
@@ -2017,6 +2017,9 @@ IEEE Int. Conf. Robotics and Automation (ICRA), 2021. [PDF](http://rpg.ifi.uzh.c
 - <a name="Ayyad21arxiv"></a>Ayyad, A., Halwani, M., Swart, D., Muthusamy, R., Almaskari, F., Zweiri, Y.,   
 *[Neuromorphic Vision Based Control for the Precise Positioning of Robotic Drilling Systems](https://arxiv.org/pdf/2201.01626)*,  
 arXiv, 2021. [Video](https://drive.google.com/file/d/1q9QwPvkd7ZcEBcGMIxIVy2r_iRfTKCSe/view).
+- <a name="Wang22iros"></a> Z. Wang, F. Cladera Ojeda, A. Bisulco, D. Lee, C. J. Taylor, K. Daniilidis, M. A. Hsieh, D. D. Lee and V. Isler,
```

Comment:
Could you write the names as Wang, Z., Cladera Ojeda, F., ... thanks

Reply:
Also, need two white spaces at the end of the line(s) to force a line break. Please see the preview when writing the commit.